analyzers/src/SonarAnalyzer.Common/Helpers/ProjectConfigReader.cs

```diff
@@ -0,0 +1,80 @@
+﻿/*
```

Comment:
Is merge to the feature branch intentional on this one? I'd be in for merging it to master instead. The whole sprint would have to remain on a feature branch like this.

Reply:
..after removing the older commits from this branch

Reply:
Yes, because I would keep it on the master branch until we get it working.

Either way, this isn't consumed by anything itself... it has no added value